client/src/Header/header.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import playLogo from './Images/play_logo.png';
+import './header.style.css';
+
+class Header extends Component {
+    render() {
+        return (
+            <div className="header">
+                <ul className="nav-bar">
+                    <li className="nav-element" id="logo">
+                        <a href="https://play.google.com/store">
+                            <img src={playLogo} alt="Google Play Logo"/>
+                        </a>
+                    </li>
+                    <li className="nav-element link">
+                        <a href="https://bit.ly/30xkTrQ">
```

Comment:
direct link please. and also, lets link to the game category https://play.google.com/store/apps/category/GAME